tests/unit/conftest.py

```diff
@@ -16,46 +16,5 @@
 """
 Module for reusable pytest fixtures.
 """
-import json
 
-import pytest
-from google.oauth2 import service_account
-
-
-API_BASE_URL = 'https://example.com'
-API_URL = f'{API_BASE_URL}/v1/foo_endpoint'
-
-
-@pytest.fixture
-def fake_keyfile_data():
-    return {
-        'type': 'service_account',
-        'project_id': 'a-test-project',
-        'private_key_id': 'yeahright',
-        'private_key': 'nope',
-        'client_email': 'test-key@a-test-project.iam.gserviceaccount.com',
-        'client_id': '12345678910',
-        'auth_uri': f'{API_BASE_URL}/auth',
-        'token_uri': f'{API_BASE_URL}/token',
-        'auth_provider_x509_cert_url': f'{API_BASE_URL}/certs',
-        'client_x509_cert_url': f'{API_BASE_URL}/x509/a-test-project'
-    }
-
-
-@pytest.fixture
-def fake_keyfile(fake_keyfile_data, tmpdir):
-    tmp_keyfile = tmpdir.mkdir('keys').join('fake_keyfile.json')
-    tmp_keyfile.write(json.dumps(fake_keyfile_data))
-    return tmp_keyfile
-
-
-@pytest.fixture
-def mock_credentials(mocker, monkeypatch):
-    mock_creds = mocker.MagicMock(service_account.Credentials, autospec=True)
-    sa_creds = mocker.MagicMock(service_account.Credentials, autospec=True)
-    sa_creds._make_authorization_grant_assertion.return_value = 'deadb33f=='
-    mock_creds.from_service_account_info.return_value = sa_creds
-
-    patch = 'gordon_janitor_gcp.http_client.service_account.Credentials'
-    monkeypatch.setattr(patch, mock_creds)
-    return mock_creds
+import pytest  # NOQA
```

Comment:
I've decided to leave this file in for future shared fixture use, but I'm fine with removing it, too.

Reply:
+1 for leaving it, I'm going to use this in my PR for some other stuff.